trace/backend_collector.go

```diff
@@ -280,6 +282,45 @@ func (c *BackendCollector) processTLSHandshake(tls akinet.TLSHandshakeMetadata)
 	return nil
 }
 
+var cloudAPIEnvironmentsPathRE = regexp.MustCompile(`^/environments/[^/]+$`)
+
+// Returns true if the witness should be excluded from Repro Mode.
+//
+// XXX This is a stop-gap hack to exclude certain endpoints for Cloud API from
+// Repro Mode.
+func excludeWitnessFromReproMode(w *pb.Witness) bool {
+
+	httpMeta := w.GetMethod().GetMeta().GetHttp()
+	if httpMeta == nil {
+		return false
+	}
+
+	switch httpMeta.Host {
+	case "api.getpostman-stage.com", "api.getpostman.com":
```

Comment:
I think that unfortunately there are several more domain names that are possible than this.

I tried and `api.postman.com` worked, while `api.postman.co` got a 301 (but maybe we need to match it anyway?)

I will communicate some possible sources of truth offline.